app/graphql/concerns/validate_submission_gradable.rb

```diff
@@ -1,43 +1,59 @@
 module ValidateSubmissionGradable
   extend ActiveSupport::Concern
 
-  class OwnersShouldBeActive < GraphQL::Schema::Validator
+  class SubmissionShouldBeGradable < GraphQL::Schema::Validator
     def validate(_object, _context, value)
-      submission = TimelineEvent.find(value[:submission_id])
+      @error = nil
+      @submission = TimelineEvent.find_by(id: value[:submission_id])
 
-      #days since submission
+      submission_exists?(value[:submission_id]) && owners_active? &&
+        submission_live?
+      @error
+    end
+
+    def submission_exists?(submission_id)
+      return true if @submission.present?
+
+      @error =
+        I18n.t(
+          "mutations.create_grading.submission_missing_error",
+          submission_id: submission_id
+        )
+
+      false
+    end
+
+    def owners_active?
+      # days since submission
       days_since_submission =
-        (Time.zone.now - submission.created_at) / (3600 * 24)
+        (Time.zone.now - @submission.created_at) / (3600 * 24)
       submission_review_allowed_days =
         Rails.application.secrets.inactive_submission_review_allowed_days
 
       submission_review_allowed =
         (days_since_submission < submission_review_allowed_days)
 
-      if (submission.students.active.empty? && !submission_review_allowed)
-        return(
-          I18n.t('validate_submission_gradable.owners_should_be_active.error')
-        )
+      if !submission_review_allowed && @submission.students.active.empty?
+        @error =
+          I18n.t("validate_submission_gradable.owners_should_be_active.error")
+        return false
       end
+
+      true
     end
-  end
 
-  class SubmissionShouldBeLive < GraphQL::Schema::Validator
-    def validate(_object, _context, value)
-      submission = TimelineEvent.find(value[:submission_id])
+    def submission_live?
+      return true unless @submission.archived?
 
-      if submission.archived?
-        return(
-          I18n.t('validate_submission_gradable.submission_should_be_live.error')
-        )
-      end
+      @error = I18n.t("validate_submission_gradable.submission_should_be_live")
+
+      false
```

Comment:
Why are we returning `false` for this method 🤔; I have seen this across the PR, if methods doesn't return anythings, the check for true/false will return false